edk2toolext/omnicache.py

```diff
@@ -95,6 +95,11 @@ def _InitOmnicache(self):
         ret = RunCmd("git", "init --bare", workingdir=self.path)
         if (ret != 0):
             return ret
+
+        ret = RunCmd("git", "config --local fetch.parallel 0", workingdir=self.path)
```

Comment:
Where are the details of these two config changes documented? Would it be appropriate to have a comment here about why the cache repo must be configured this way?

Reply:
Will add a comment; the default behavior of git fetch is single threaded. fetch.parallel = 0 uses "a reasonable default" (per the git docs) - I looked into the source and presently that means a number of threads equal to current cpu count. Original implementation I set up forked a thread for each remote; this was running our build agent VMs out of memory.